packages/asc-core/src/components/Button/Button.ts

```diff
@@ -79,4 +79,20 @@ export const IconButton = styled(ButtonBase)<Props>`
   padding: 5px;
 `
 
+export const SocialButton = styled(ButtonBase)<Props>`
+  padding: 0px;
+  width: 30px;
```

Comment:
Shouldn't this value be configurable as a component prop?
Also, the `ButtonBase` component doesn't have styling for `:focus`. Shouldn't the focus state styling be defined here for the `SocialButton` component?

Reply:
Added :focus. For now the social button just reuses the styling and should always be 30px. Can be changed later on with a default of 30 if needed